website/docs/FAQ.md

```diff
@@ -127,3 +127,7 @@ Workaround:
 2. `mkdir /home/vscode/.local/lib/python3.10/site-packages/google/colab`
 
 Explanation: Per [this gist](https://gist.github.com/defulmere/8b9695e415a44271061cc8e272f3c300?permalink_comment_id=4711478#gistcomment-4711478), linked from the official [chromadb docs](https://docs.trychroma.com/troubleshooting#sqlite), adding this folder triggers chromadb to use pysqlite3 instead of the default.
+
+## How to get last message ?
+
+Please refer to https://microsoft.github.io/autogen/docs/reference/agentchat/conversable_agent#chat_messages
```

Comment:
The question and answer don't match. There are two methods, one for getting last message and the other for getting all the messages.

Reply:
Thanks, changed made to last message